docs/chapter4-payload-handoff-format.rst

```diff
@@ -29,9 +29,833 @@ architecture, either 32 or 64 bits.
+Many of these sections refer to the Devicetree Specification [DTspec]_.
 
-TBD (Nodes)
+4.1 Standard Properties and Property Values
+---------------------------------------------
+
+4.1.1 Property Values
+~~~~~~~~~~~~~~~~~~~~~
+
+A property value is an array of zero or more bytes that contain information
+associated with the property.
+
+Properties might have an empty value if conveying true-false information. In
+this case, the presence or absence of the property is sufficiently descriptive.
+
+Here are list of standard value types defined by the DTSpec [DTspec]_:
+
+.. tabularcolumns:: | p{4cm} p{12cm} |
+.. _property-values-table:
+.. table:: Property values
+   :class: longtable
+
+   ======================== ==================================================================
+   Value                    Description
+   ======================== ==================================================================
+   ``<empty>``              Value is empty. Used for conveying true-false information, when
+                            the presence or absence of the property itself is sufficiently
+                            descriptive.
+   ``<u32>``                A 32-bit integer in big-endian format. Example: the 32-bit value
+                            0x11223344 would be represented in memory as:
+
+                               ::
+
+                                  address    11
+                                  address+1  22
+                                  address+2  33
+                                  address+3  44
+   ``<u64>``                Represents a 64-bit integer in big-endian format. Consists of
+                            two ``<u32>`` values where the first value contains the most
+                            significant bits of the integer and the second value contains
+                            the least significant bits.
+
+                            Example: the 64-bit value 0x1122334455667788 would be
+                            represented as two cells as: ``<0x11223344 0x55667788>``.
+
+                            The value would be represented in memory as:
+
+                               ::
+
+                                    address  11
+                                  address+1  22
+                                  address+2  33
+                                  address+3  44
+                                  address+4  55
+                                  address+5  66
+                                  address+6  77
+                                  address+7  88
+   ``<string>``             Strings are printable and null-terminated. Example: the string
+                            "hello" would be represented in memory as:
+
+                               ::
+
+                                    address  68  'h'
+                                  address+1  65  'e'
+                                  address+2  6C  'l'
```

Comment:
6c

(should use lower-case hex throughout)

Reply:
done.